backend/app/views/spree/admin/shared/_sidebar_left.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="sidebar">
+  <%= render partial: 'spree/admin/shared/sidebar_left_header' %>
+  <%= render partial: 'spree/admin/shared/menu' %>
+  <%= render partial: 'spree/admin/shared/sidebar_left_footer' %>
```

Comment:
Maybe now is a good time to move all sidebar related stuff into it's own folder `spree/admin/sidebar`?

```
spree/admin/sidebar
  |-- _header.html.erb
  |-- _menu.html.erb
  |-- _footer.html.erb
```